src/test/mir-opt/address_of.borrow_and_cast.SimplifyCfg-initial.after.mir

```diff
@@ -38,7 +38,7 @@ fn borrow_and_cast(_1: i32) -> () {
         _6 = &raw mut (*_7);             // scope 2 at $DIR/address-of.rs:44:13: 44:19
         FakeRead(ForLet, _6);            // scope 2 at $DIR/address-of.rs:44:9: 44:10
         StorageDead(_7);                 // scope 2 at $DIR/address-of.rs:44:31: 44:32
-        _0 = const ();                   // scope 0 at $DIR/address-of.rs:41:32: 45:2
+        _0 = ();                         // scope 0 at $DIR/address-of.rs:41:32: 45:2
```

Comment:
There is a difference between these two. `const ()` is a `Constant`, while `()` is an `Aggregate`.